### ISSUE TITLE
Checking refinement stops with error "identifier ... is either undefined or not an operator" (since toolbox version 1.8.0)

### ISSUE THREAD

**Issue by matthias-g** [tlaplus/tlaplus#817]
## Description
When checking that specification A refines specification B, TLC reports an error if specification B uses an INSTANCE statement.
The bug exists in version 1.8.0 of the toolbox but the bug does not exist in version 1.7.1 of the toolbox.

As a simple example, I created the specification in https://github.com/matthias-g/cherrytree.
In this example, TLC should verify that a cherry tree is a tree.
In version 1.8.0 of the toolbox, TLC reports an error when trying to check that `CherryTree` refines `Tree`. However, checking that `CherryTree` refines `FruitTree` succeeds.
In version 1.7.1 of the toolbox, both refinements can be successfully checked.

## Expected Behavior
Checking the model `Refinement` is successful.

## Actual Behavior
TLC reports the following error:

```
The error occurred when TLC was evaluating the nested
expressions at the following positions:
0. Line 20, column 16 to line 20, column 19 in Tree
1. Line 14, column 5 to line 14, column 21 in Tree
2. Line 14, column 8 to line 14, column 21 in Tree
3. Line 8, column 1 to line 8, column 31 in Tree
4. Line 16, column 5 to line 16, column 17 in FruitTree
5. Line 16, column 8 to line 16, column 17 in FruitTree
6. Line 9, column 5 to line 10, column 29 in FruitTree
7. Line 9, column 8 to line 9, column 26 in FruitTree
8. Line 10, column 8 to line 10, column 29 in FruitTree
9. Line 10, column 8 to line 10, column 18 in FruitTree
10. Line 10, column 8 to line 10, column 17 in FruitTree
11. Line 17, column 19 to line 17, column 38 in CherryTree
12. Line 12, column 5 to line 14, column 19 in CherryTree
13. Line 12, column 8 to line 12, column 42 in CherryTree
14. Line 12, column 8 to line 12, column 17 in CherryTree


TLC threw an unexpected exception.
This was probably caused by an error in the spec or model.
See the User Output or TLC Console for clues to what happened.
The exception was a java.lang.RuntimeException
: 
In evaluation, the identifier FruitState is either undefined or not an operator.
line 12, col 8 to line 12, col 17 of module CherryTree
```

## Steps to Reproduce
1. Download https://github.com/matthias-g/cherrytree
2. Open spec `CherryTree` in the toolbox
3. Check model `Refinement`

## Your Environment
 - Toolbox version: 1.8.0
 - Operating System: Ubuntu 22.04

**Comment by lemmy** [comment]
Regression caused by https://github.com/tlaplus/tlaplus/issues/798

**Comment by Calvin-L** [comment]
@matthias-g This should be fixed as of commit dfbf8cea (PR #800). Let us know if you have further trouble. :)
